docs/docs-beta/docs/tutorial/pinecone.md

```diff
@@ -0,0 +1,311 @@
+---
```

Comment:
Should we standardize project naming convention?

Modal and Bluesky are named `examples/project_` whereas this is `examples/tutorial_`.